packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -251,30 +254,30 @@ export default class Calendar extends React.Component {
       return
     }
 
-    let classes = ['react-datepicker__navigation', 'react-datepicker__navigation--next']
+    let classes = ['react-datepicker-navigation', 'react-datepicker-navigation--next']
     if (this.props.showTimeSelect) {
-      classes.push('react-datepicker__navigation--next--with-time')
+      classes.push('react-datepicker-navigation--next--with-time')
     }
     if (this.props.todayButton) {
-      classes.push('react-datepicker__navigation--next--with-today-button')
+      classes.push('react-datepicker-navigation--next--with-today-button')
     }
 
     return <a
-      className={classes.join(' ')}
+      className={cx(classes)}
       onClick={this.increaseMonth} />
   }
 
   renderCurrentMonth = (date = this.state.date) => {
-    const classes = ['react-datepicker__current-month']
+    const classes = ['react-datepicker-current-month']
 
     if (this.props.showYearDropdown) {
-      classes.push('react-datepicker__current-month--hasYearDropdown')
+      classes.push('react-datepicker-current-month--hasYearDropdown')
     }
     if (this.props.showMonthDropdown) {
-      classes.push('react-datepicker__current-month--hasMonthDropdown')
+      classes.push('react-datepicker-current-month--hasMonthDropdown')
     }
     return (
-      <div className={classes.join(' ')}>
+      <div className={cx(classes.join(' '))}>
```

Comment:
I dont think we need to "join" the array elements, since "cx" accepts arrays for multiple class names.